client/src/containers/NewMealForm.js

```diff
@@ -163,10 +163,22 @@ export const validate = values => {
     errors.price = 'Required';
   }
 
+  if (isNaN(values.price)) {
+    errors.price = 'Price should be a number';
+  }
+
   if (!values.servings) {
     errors.servings = 'Required';
   }
 
+  if (isNaN(values.servings)) {
+    errors.servings = 'Servings should be a number';
+  }
+
+  if (isNaN(values.zipcode)) {
+    errors.zipcode = 'Zipcode should be a number';
```

Comment:
Zipcode can sometimes be specified with four additional numbers like 99750-0077. Up to you if you want to change the validation to account for this.